### PR TITLE
Feature/#77 allow user to remove block

### PR DESCRIPTION
**Pull request by dobi28**

### CODE REVIEW

File: WebAPI/Controllers/StudentController.cs [hunk 1]
```diff
@@ -70,10 +73,13 @@ public async Task<IActionResult> AddNewBlock([FromBody]string studentId, [FromBo
             return Ok(block);
         }
 
-
         [HttpPost]
-        public async Task<IActionResult> RemoveBlock([FromBody]string studentId, [FromBody]Block block)
+        [HttpPost("removeBlock")]
```
Comment (ZVARKO) [inline WebAPI/Controllers/StudentController.cs]: There is HttpPost twice

File: WebApp/src/actions/timetableActions.js [hunk 2]
```diff
@@ -125,5 +128,42 @@ export function hideCourseTimetable(course) {
...
+    Day: course.day,
+    Teacher: course.teacher,
+    Room: course.room,
+    StartHour: course.startBlock + 6,
+    Duration: course.endBlock - course.startBlock,
+    Type: ((course.type == 'laboratory') ? (2) : (3))
+  }
+
+  const body = {
+    studentId: "invalidID",
```
Comment (ZVARKO) [inline WebApp/src/actions/timetableActions.js]: invalidID???

File: WebAPI/Models/RemoveBlockModels/RemoveRequestModel.cs [hunk 3]
```diff
@@ -0,0 +1,12 @@
+using WebAPI.Models.RemoveBlockModel;
+
+namespace WebAPI.Models.RemoveBlockModels
+{
+    public class RemoveRequestModel
+    {
+        public string StudentId { get; set; }
```
Comment (ZVARKO) [inline WebAPI/Models/RemoveBlockModels/RemoveRequestModel.cs]: Use data annotations to validate models

File: WebAPI/Models/RemoveBlockModels/RemoveRequestModel.cs [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+namespace WebAPI.Models.RemoveBlockModels
+{
+    public class RemoveRequestModel
+    {
+        [Required(ErrorMessage = "StudentID je povinné.")]
+        public string StudentId { get; set; }
+        
+        public RemovedBlockModel RemoveBlock { get; set; }
+
+        public RemoveRequestModel() { }
```
[the remark this answers is not in the document]
Comment (dobi28) [reply WebAPI/Models/RemoveBlockModels/RemoveRequestModel.cs]: fix

File: WebAPI/Controllers/StudentController.cs (hunk 1 above)
Comment (dobi28) [reply WebAPI/Controllers/StudentController.cs]: removed one post

File: WebAPI/Models/RemoveBlockModels/RemoveRequestModel.cs (hunk 3 above)
Comment (dobi28) [reply WebAPI/Models/RemoveBlockModels/RemoveRequestModel.cs]: fiix

**Pull request merged**